rotkehlchen/tests/unit/globaldb/test_prices.py

```diff
@@ -1,5 +1,13 @@
-from rotkehlchen.constants.assets import A_BAL, A_BTC, A_ETH, A_USD
+
+import pytest
```

Comment:
```suggestion
import pytest
```

Don't leave empty newline at start of the file. Guess editor did it.